game-app/package-lock.json

```diff
@@ -16,6 +16,7 @@
         "@headlessui/react": "^1.7.17",
         "@latticexyz/recs": "^1.43.0",
         "@latticexyz/utils": "^1.43.0",
+        "@react-spring/web": "^9.7.3",
```

Comment:
I don't know why the lock was affected, I simply cloned the repo and ran `npm i` at the root of game-app